test/unit/annotation_layer_spec.js

```diff
@@ -179,8 +179,8 @@ describe('Annotation layer', function() {
     var annotations;
 
     beforeEach(function(done) {
-      var pdfUrl = combineUrl(window.location.href,
-                              '../pdfs/annotation-fileattachment.pdf');
+      var pdfUrl = new URL('../pdfs/annotation-fileattachment.pdf', 
+                            window.location).href;
```

Comment:
Nit: please reduce the indentation with *one* space, so that it's lined up like this:
```javascript
      var pdfUrl = new URL('../pdfs/annotation-fileattachment.pdf', 
                           window.location).href;
```